python/lsst/obs/subaru/gen3/hsc/rawFormatter.py

```diff
@@ -43,7 +44,10 @@ def makeVisitInfo(self, metadata):
         maker = MakeHscRawVisitInfo()
         return maker(metadata)
 
-    def makeWcs(self, metadata):
+    def getDetector(self, id):
+        return HyperSuprimeCam().getCamera()[id]
+
+    def _createSkyWcsFromMetadata(self, metadata):
```

Comment:
Please add a short docstring, perhaps mentioning why `makeFlippedWcs` is needed here instead of `makeSkyWcs`

Reply:
I'll add a comment to that effect. The docstring of the method comes from the parent class.